### ISSUE TITLE
Absent alignment requirements for the CONTEXT struct 

### ISSUE THREAD

**Issue by HoShiMin** [microsoft/windows-rs#1955]
### Which crate is this about?

windows, windows-sys

### Summary

According to the definition in the Windows.h, the CONTEXT struct must be aligned by 16 bytes, but it is not neither in windows_rs, nor in windows_sys (it's just a #[repr(C)] here).
It causes an ERROR_NOACCESS errors in the GetThreadContext()/SetThreadContext() functions.

**Comment by riverar** [comment]
Agree, there's a higher level packing/architecture bug here in metadata, transferring to that repo.

Related: 
* https://github.com/microsoft/win32metadata/issues/460
* https://github.com/microsoft/win32metadata/issues/870
